src/Websites/Websites/Cmdlets/WebApps/PublishAzureWebApp.cs

```diff
@@ -72,6 +75,11 @@ public override void ExecuteCmdlet()
                 using (var s = File.OpenRead(ArchivePath))
                 {
                     HttpClient client = new HttpClient();
+                    if (Timeout != 0)
+                    {
+                        // Considering the deployment of large packages the default time(150 seconds) is not sufficient. So increased the timeout based on user choice.
+                        client.Timeout = TimeSpan.FromMilliseconds(Timeout);
+                    }
```

Comment:
the time out should have a low bound or min value where the min value should  be the default timeout value ; right now seems like folks have can set time out to 1ms?

Reply:
Taken care!